packages/components/src/Placeholder.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsxImportSource theme-ui */
```

Comment:
Could you make a couple of stories for this and add this under StoryBook `Elements` (where the `Buttons` are) @achrafismyname ? thanks!